### PR TITLE
aws-es-proxy 0.8 (new formula)

### PR DESCRIPTION
**Pull request by dlackty**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/aws-es-proxy.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+class AwsEsProxy < Formula
+  desc "Small proxy between HTTP client and AWS Elasticsearch"
+  homepage "https://github.com/abutaha/aws-es-proxy"
+  url "https://github.com/abutaha/aws-es-proxy/archive/v0.8.tar.gz"
+  sha256 "f1462dcfd75aa2a8e7e370f2cfac3241bcc44de3406181fc164b9c1f90315f2c"
+  head "https://github.com/abutaha/aws-es-proxy.git"
```
Comment (fxcoudert) [inline Formula/aws-es-proxy.rb]: No head for new formulas

File: Formula/aws-es-proxy.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  def install
+    ENV["GLIDE_HOME"] = buildpath/"glide_home"
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/abutaha/aws-es-proxy").install buildpath.children
+    cd "src/github.com/abutaha/aws-es-proxy" do
+      system "glide", "install"
+      system "go", "build", "-o", "aws-es-proxy"
+
+      bin.install "aws-es-proxy"
```
Comment (fxcoudert) [inline Formula/aws-es-proxy.rb]: No man, no doc?

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/aws-es-proxy.rb (hunk 1 above)
Comment (dlackty) [reply Formula/aws-es-proxy.rb]: Removed.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Thanks @dlackty for the pull request!

**Comment by dlackty** [comment]
Thanks @fxcoudert and you mahjong avatar is cool!

**Pull request closed without merging**